### DISCUSSION TITLE
Evolution 1 with USR-W610

### DISCUSSION THREAD

**Discussion by ChuckWhitson** [jgyates/genmon#1173] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open issues](https://github.com/jgyates/genmon/issues),[closed issues](https://github.com/jgyates/genmon/issues?q=is%3Aissue+is%3Aclosed),[open discussion](https://github.com/jgyates/genmon/discussions), [closed discussion](https://github.com/jgyates/genmon/discussions/categories/questions?discussions_q=is%3Aclosed) - which may answer my question or that I could contribute my information to.
- [X] I have read the [Known Issues Wiki Page](https://github.com/jgyates/genmon/wiki/Appendix-D-Known-Issues) and my question / issue is not listed.
- [X] I have provided the version of the software and the operating system in my post.
- [X] If I have an issue with genmon not communicating with the controller, I have read and followed the [serial troubleshooting guide](https://github.com/jgyates/genmon/wiki/3.6---Serial-Troubleshooting).
- [X] I have included the generator make and model in my submission.

### Version

1.19.02

### Have you submitted your log files from the About page? Note that outbound email must be working for logs to be submitted.

I have not submitted logs

### What is/are your question(s)?

I have a Generac model G0070431 with an Evolution 1 controller. I plan to use a USR-W610 Serial to WiFi Ethernet Wireless Converter outside with genmon running on Debian inside the house. The USR-W610 came with zero documentation and what I’ve found online really only seems to provide directions on how to connect it to your WiFi network and thats about it which I’ve already done. I’m wondering if there is anyone out there that has this setup or something similar, and if so could you help me out with the config in genmon and the USR-W610.

**Comment by skipfire** [comment]
I have the Ethernet USR, which hopefully only differs by network which you have already handled.

To enable the connection you need to set the USR up as a Serial Server, in there you also get a network port option, I believe mine defaulted to port 8235.

Next in GenMon you turn on `Enable Serial over TCP/IP` and then enter the USR's IP and the port selected in the USRs serial server and save.  You may later want to adjust some GenMon settings to reduce false alarms that come with Serial over TCP solutions.

The other component is cabling, you can either follow the [making a cable wiki](https://github.com/jgyates/genmon/wiki/3.1--Making-a-Cable) or you can buy the PintSize Molex Generator to Serial Device adapter (I'm the guy behind PintSize).

If you are stuck on one of the steps let me know and I can hook up my USR to get specific configurations I did.

**Comment by jgyates** [comment]
This thread has some info on the W610 setup:  https://github.com/jgyates/genmon/issues/863

Also, you can search past issues and discussion with the "Serial over TCP" label for more threads on this topic: https://github.com/jgyates/genmon/issues?q=label%3A%22Serial+over+TCP%22+

**Reply by jgyates** [reply]
and the wiki has info on setting up genmon with serial over IP: https://github.com/jgyates/genmon/wiki/Appendix-F----Serial-over-IP
